.jenkins/pytorch/common.sh

```diff
@@ -139,18 +139,18 @@ fi
 # Linux bionic cannot find conda mkl with cmake 3.10, so we need a cmake from conda.
 # Alternatively we could point cmake to the right place
 # export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
-if [[ "$BUILD_ENVIRONMENT" == *pytorch-xla-linux-bionic* ]] || \
-   [[ "$BUILD_ENVIRONMENT" == *pytorch-linux-xenial-cuda9-cudnn7-py2* ]] || \
-   [[ "$BUILD_ENVIRONMENT" == *pytorch-linux-xenial-cuda10.1-cudnn7-py3* ]] || \
-   [[ "$BUILD_ENVIRONMENT" == *pytorch-*centos* ]] || \
-   [[ "$BUILD_ENVIRONMENT" == *pytorch-linux-bionic* ]]; then
+if [[ "$BUILD_ENVIRONMENT" == *xla-linux-bionic* ]] || \
+   [[ "$BUILD_ENVIRONMENT" == *linux-xenial-cuda9-cudnn7-py2* ]] || \
+   [[ "$BUILD_ENVIRONMENT" == *linux-xenial-cuda10.1-cudnn7-py3* ]] || \
+   [[ "$BUILD_ENVIRONMENT" == *centos* ]] || \
+   [[ "$BUILD_ENVIRONMENT" == *linux-bionic* ]]; then
```

Comment:
should we have "another" common script to define these comparisons, which will help with code readability and maintainability. Aslo *  shorten the code, but make it harder to understand and change (if needed)

Reply:
I see your point, but these are pretty sparse and changes a lot. I think we need a more comprehensive refactoring. I will keep it for now.